src/frontend/src/lib/components/tokens/TokenCard.svelte

```diff
@@ -17,7 +19,12 @@
 		aria-label={`Open the list of ${token.symbol} transactions`}
 	>
 		<Card noMargin>
-			{token.name}
+			{#if isTokenCkLedger(token)}
+				<Tag>ck</Tag>
+				{token.twinToken?.name ?? token.name.replace(/^ck/, '')}
```

Comment:
I'm not a fan of adding an exception for ck tokens within a generic component. Is there a more generic, maintainable and future proof way to approach this rather than hardcoding an exception?

Reply:
we could create a `displayName` prop for the class `Token` and set it differently for the ck tokens, but my issue was with the Tag component

unless we make `displayName: string | SvelteComponent`